packages/central-server/src/tests/apiV2/import/importSurveyResponses/testFunctionality.js

```diff
@@ -110,8 +110,14 @@ export const testFunctionality = async () => {
     await app.grantFullAccess();
 
     await findOrCreateDummyCountryEntity(models, { code: 'DL', name: 'Demo Land' });
-    const entities = ['DL_1', 'DL_5', 'DL_7', 'DL_9'].map(code => ({
-      code,
+    const entities = [
+      { code: 'DL_1', name: 'Port Douglas' },
+      { code: 'DL_5', name: 'Hawthorn East' },
+      { code: 'DL_7', name: 'Lake Charm' },
+      { code: 'DL_9', name: 'Thornbury' },
+    ].map(entity => ({
+      code: entity.code,
+      name: entity.name,
```

Comment:
```suggestion
    ].map(entity => ({
      ...entity,
```

Nice case for a spread here